compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -553,7 +551,7 @@ private BLangFunction createGeneratedInitializerFunction(BLangClassDefinition cl
         }
 
         return wireUpGeneratedInitFunction(generatedInitFunc,
-                    (BObjectTypeSymbol) classDefinition.symbol, classDefinition.initFunction);
```

Comment:
But those are valid formatting changes ryt?
In default those formatting changes done by IntelliJ.

Reply:
Not always, I guess. Since it would depend on a person's local settings. It complicates reviewing a bit too (unless of course reviewers opt to hide whitespace and review). 

In general, I think it's a good practice to not format unrelated areas in a PR send for some other fix/purpose. If we need to format like an entire file, we can send a separate PR.

Reply:
ok. will update.

Reply:
Updated